check_galera_cluster

```diff
@@ -43,6 +43,29 @@ print_help() {
 crit=2
 fcp=0.1
 
+#check required tools
+check_executable() {
+
+    if [ -z "$1" ]; then
+        echo "check_executable: no parameter given!"
+        exit $ST_UK
+    fi
+
+    if [ -z "$(which $1)" ]; then
+            echo "UNKNOWN: Cannot find $1"
+            exit $ST_UK
+    fi
+
+}
+check_executable mysql
+check_executable bc
```

Comment:
always good to have.
Can you use `command -v "$1"` instead of `[ -z "$(which $1)"]` ?   `command -v` returns 0 if found, 1 otherwise, and it's a builtin bash command.
```bash
if !command -v "$1" > /dev/null; then
  echo "UNKNOWN: Cannot find $1"
  exit $ST_UK
fi
```
should be enough, no need to check $1 in your function since you're only using it twice just after the definition, it's not like it's gonna be used elsewhere.